README.md

```diff
@@ -282,8 +282,8 @@ net.core.netdev_max_backlog=
 
 ## Contributing
 
-Please see the [Juju SDK docs](https://juju.is/docs/sdk) for guidelines on enhancements to this charm following best practice guidelines, and [CONTRIBUTING.md](https://github.com/canonical/kafka-operator/blob/main/CONTRIBUTING.md) for developer guidance.
-
+Please see the [Juju SDK docs](https://juju.is/docs/sdk) for guidelines on enhancements to this charm following best practice guidelines, and [CONTRIBUTING.md](https://github.com/canonical/kafka-operator/blob/main/CONTRIBUTING.md) for developer guidance. 
+Also, if you truly enjoy working on open-source projects like this one and you would like to be part of the OSS revolution, please don't forget to check out the [open positions](https://canonical.com/careers/all) we have at [Canonical](https://canonical.com/). 
```

Comment:
I would put this right at the top of the readme.

Reply:
I'm inclined to think that the first paragraphs at the top should be about the product, also following the standard format for the page [here](https://docs.google.com/document/d/1Zw-UoQzHMSQjATLXjjvd9GKu5qU2mHmqxtKck8nGkbU/edit). This bit is more related to Canonical, who is building the product, but should go a bit below. Contributing section felt an appropriate place. We could however give this a bit more relevant by using a box (e.g. using quoting), like:

> if you truly enjoy working on open-source projects like this one and you would like to be part of the OSS revolution, please don't forget to check out the [open positions](https://canonical.com/careers/all) we have at [Canonical](https://canonical.com/).

Reply:
I think it's good to keep here too!